deploy/helm/sumologic/templates/_helpers/_logs/_receivers.tpl

```diff
@@ -0,0 +1,253 @@
+{{/*
```

Comment:
Instead of putting this in a tpl file, wouldn't it be better to have it in a templated .yaml file and include that file where needed? This is a bit hard to read.

Reply:
It is not, as path regex inside OTELcol is different for windows and for linux

Reply:
Alternatively I can write regex to handle botf formats. Would it that be ok?

Reply:
I'm not sure I understand. If we used a templated yaml file, we could still use different formats for windows and linux, right?

Reply:
I don't understand. How does this additional file will know if it should be windows or linux template?